File: app/src/main/java/com/travlog/android/apps/libs/utils/StringUtils.java
package com.travlog.android.apps.libs.utils;

import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.util.Patterns;

public final class StringUtils {

    private StringUtils() {
    }

    public static boolean isEmail(final @NonNull CharSequence str) {
        return Patterns.EMAIL_ADDRESS.matcher(str).matches();
    }

    public static boolean isEmpty(final @Nullable String str) {
        return str == null || str.trim().length() == 0;
    }
}

File: app/src/main/java/com/travlog/android/apps/libs/utils/StringUtils.kt
package com.travlog.android.apps.libs.utils

import android.util.Patterns

object StringUtils {

    fun isEmail(str: CharSequence): Boolean {
        return Patterns.EMAIL_ADDRESS.matcher(str).matches()
    }

    fun isEmpty(str: String): Boolean {
        return str.trimIndent().length == 0
    }
}
